lib/chef/resource/openssl_x509_certificate.rb

```diff
@@ -109,30 +109,41 @@ class OpensslX509Certificate < Chef::Resource
       property :ca_key_pass, String,
         description: "The passphrase for CA private key's passphrase."
 
+      property :renew_before_expiry, Integer,
+        description: "The number of days before the expiry. The certificate will be automaticaly renewed when the value is reached."
```

Comment:
we'll want to get an introduced field added here so this shows up in our docs correctly:
```ruby
introduced: 15.7
```

Reply:
Done.